### PR TITLE
Fix #3683: Allow Admins to delete a user from Users page

### LINKED ISSUE #3668: On refreshing users page, No users added is displayed before all the users are displayed on the screen
**Affected module**
UI

**Describe the bug**
On refreshing users page, No users added is displayed before all the users are displayed on the screen

**To Reproduce**
- Navigate to Users page from Settings
- Refresh the page 
- "**No users added**" message is displayed before all the users are displayed on the screen

https://user-images.githubusercontent.com/67048554/160129996-b4516bec-84f4-42a3-9bf7-087d5e046f9c.mov

**Expected behaviour**
No Users added message should not be displayed

**Version:**
 - OpenMetadata version: 0.9.1

**Additional context**
NA

### LINKED ISSUE #3683: Allow Admins or permitted users to delete a user from Users page
**Is your feature request related to a problem? Please describe.**
Being an admin, do not have any option to delete a user from Users page

**Describe the solution you'd like**
Provide Admin only access to delete any user from Users page

**Describe alternatives you've considered**
If user can have standalone permission to delete other users, provide support for Role Based Access as well

**Additional context**
Restrict self-deletion of user, or handle it properly if allowed

### PR DESCRIPTION
**Pull request by darth-coder00**
Closes #3683 
Closes #3668
### Describe your changes :

I worked on the Users page to allow deletion and handle error cases

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

https://user-images.githubusercontent.com/86726556/160666449-63cba702-1bdd-47a8-b0d9-fcfb4eab70f6.mov

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@harshach @vivekratnavel @Sachin-chaurasiya @shahsank3t

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/jsons/en.ts [hunk 1]
```diff
@@ -27,11 +27,12 @@ const jsonData = {
 
     'delete-glossary-error': 'Error while deleting glossary!',
     'delete-glossary-term-error': 'Error while deleting glossary term!',
-    'delete-team-error': 'Error while deleting team!',
+    'delete-ingestion-error': 'Error while deleting ingestion workflow',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/jsons/en.ts:30]:
```suggestion
    'delete-ingestion-error': 'Error while deleting ingestion workflow!',
```

File: openmetadata-ui/src/main/resources/ui/src/jsons/en.ts [hunk 2]
```diff
@@ -57,23 +59,24 @@ const jsonData = {
...
     'update-description-error': 'Error while updating description!',
     'update-entity-error': 'Error while updating entity!',
-    'update-team-error': 'Error while updating team',
-    'update-tags-error': 'Error while updating tags!',
-    'update-task-error': 'Error while updating tasks!',
     'update-entity-follow-error': 'Error while following entity!',
     'update-entity-unfollow-error': 'Error while unfollowing entity!',
+    'update-glossary-term-error': 'Error while updating glossary term!',
     'update-ingestion-error': 'Error while updating ingestion workflow',
+    'update-owner-error': 'Error while updating owner',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/jsons/en.ts:74]:
```suggestion
    'update-owner-error': 'Error while updating owner!',
```

File: openmetadata-ui/src/main/resources/ui/src/jsons/en.ts [hunk 3]
```diff
@@ -57,23 +59,24 @@ const jsonData = {
...
     'update-service-config-error': 'Error while updating ingestion workflow',
+    'update-tags-error': 'Error while updating tags!',
+    'update-task-error': 'Error while updating tasks!',
+    'update-team-error': 'Error while updating team',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/jsons/en.ts:78]:
```suggestion
    'update-team-error': 'Error while updating team!',
```

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM, a few minor comments.

**Review by harshach — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/jsons/en.ts (hunk 1 above)
Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/jsons/en.ts:30]: This error message has some dynamic suffixes

**Pull request merged**